ci/cron/src/Main.hs

```diff
@@ -341,26 +341,35 @@ push_to_gcp gcp_credentials bash_lib local_path remote_path = do
         "gcs \"$GCRED\" cp \"" <> local_path <> "\" \"" <> remote_path <> "\"",
         "'"]
 
-check_releases :: String -> String -> IO ()
-check_releases gcp_credentials bash_lib = do
-    releases <- fetch_gh_paginated "https://api.github.com/repos/digital-asset/daml/releases"
+check_releases :: Maybe String -> String -> Maybe Int -> IO ()
+check_releases gcp_credentials bash_lib max_releases = do
+    releases' <- fetch_gh_paginated "https://api.github.com/repos/digital-asset/daml/releases"
+    let releases = case max_releases of
```

Comment:
I guess cutting of within the pagination already is not worth the complexity since it’s fast enough?

Reply:
Exactly.